scripts/base_network.py

```diff
@@ -492,6 +492,10 @@ def base_network(
     lines_ac = lines[lines.tag_frequency.astype(float) != 0].copy()
     lines_dc = lines[lines.tag_frequency.astype(float) == 0].copy()
 
+    # avoiding the conflicts in processing the network model
```

Comment:
Instead of dropping tag_frequency here, we could drop it in build_osm_nwtwork and rely on the "dc" column in base_networl instead.
What do you think?

In build osm, we have both the dc and tag_frequency columns that are passed to base_network.
Probably, we can drop tag frequency in build and rely on dc column

Reply:
Ahh, I see! We now have calculation of `dc` column directly in `clean_osm` script, so it's basically a duplication in `base_network`! Agree, no need then to keep `tag_frequency` up to `base_network`. Thanks a lot for checking it!

Revised 🙂